### PR TITLE
Altpay

### PR DESCRIPTION
**Pull request by vishakha94**

### CODE REVIEW

File: test/screenshot/config.js [hunk 1]
```diff
@@ -324,3 +328,216 @@ for (let color of getButtonConfig('paypal', 'colors')) {
         }
     });
 }
+
+
+/** INSTALLMENT BUTTON CONFIG **/
+
+for (let locale of INSTALLMENT_BTN_LOCALE) {
```
Comment (bluepnume) [inline test/screenshot/config.js]: So, one thing about these tests is they're really slow. So maybe let's not test every locale and combination. Can we, for now, maybe just render two buttons -- one with installment period and one without?

Also can we get one button for each of the new logos?

File: test/tests/button/validation.js [hunk 2]
```diff
@@ -124,6 +129,44 @@ let buttonConfigs = [
         }))
     },
 
+    {
```
Comment (bluepnume) [inline test/tests/button/validation.js]: Can we add some tests for the new logos?

File: test/screenshot/config.js [hunk 3]
```diff
@@ -323,4 +320,274 @@ for (let color of getButtonConfig('paypal', 'colors')) {
...
+            style:  {
+                color,
+                layout:     'vertical',
+                maxbuttons: 4
+            },
+            funding: {
+                allowed: [ FUNDING.MYBANK ]
+            }
+        }
+    });
```
Comment (bluepnume) [inline test/screenshot/config.js]: This all looks great 👍

File: test/screenshot/config.js (hunk 1 above)
Comment (vishakha94) [reply test/screenshot/config.js]: ok

File: test/tests/button/validation.js (hunk 2 above)
Comment (vishakha94) [reply test/tests/button/validation.js]: they are checked in now

**Pull request merged**